pwa/src/pages/test.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from "react";
+import APIService from "../apiService/apiService";
+import APIContext from "../apiService/apiContext";
+
+const ExamplePage = (props: any) => {
+  const [notifications, setNotifications] = useState<any>(null);
+  const API: APIService = React.useContext(APIContext);
+
+  useEffect(() => {
+    API.Notification.getAll().then((res) => {
+      setNotifications(res);
+    });
```

Comment:
Might be good to also document the `.catch((err) => ...` here, because its an important step of implementing the API Service.

Reply:
Note: should result in `throw new Error(err);`